README.md

```diff
@@ -16,46 +16,69 @@ Great for:
 What is this?
 ---
 
-k3s is a fully compliant Kubernetes distribution with the following changes:
-
-1. Packaged as a single binary.
-1. Lightweight storage backend based on sqlite3 as the default storage mechanism. etcd3, MySQL, Postgres also still available.
-1. Wrapped in simple launcher that handles a lot of the complexity of TLS and options.
-1. Secure by default with reasonable defaults for lightweight environments.
-1. Minimal to no OS dependencies (just a sane kernel and cgroup mounts needed). k3s packages required
-   dependencies
-    * containerd
-    * Flannel
-    * CoreDNS
-    * CNI
-    * Host utilities (iptables, socat, etc)
-    * Ingress controller (traefik)
-    * Embedded service loadbalancer
-    * Embedded network policy controller
+K3s is a [fully conformant](https://github.com/cncf/k8s-conformance/pulls?q=is%3Apr+k3s) Kubernetes distribution with the following changes:
+
+1. It is packaged as a single binary.
+1. It swaps out etcd for a lightweight sqlite3 storage backend. etcd3, MySQL, and Postgres are also supported.
+1. It wraps Kubernetes and other components in a single, simple launcher.
+1. It is secure by default with reasonable defaults for lightweight environments.
+1. It has minimal to no OS dependencies (just a sane kernel and cgroup mounts needed).
```

Comment:
Maybe also call out the simplified server-agent connectivity? Requires only a single open port on servers, no ports on agents (not counting whatever you need for CNI of course).